ostree-simplified-installer.sh

```diff
@@ -36,6 +36,11 @@ sudo cp files/fdo/rendezvous-server.yml /etc/fdo/rendezvous-server.conf.d/
 sudo cp files/fdo/serviceinfo-api-server.yml /etc/fdo/serviceinfo-api-server.conf.d/
 
 # Install yq to modify service api server config yaml file
+# Workaround - https://issues.redhat.com/browse/RHEL-21528
+if [[ "$ID" == "rhel" ]] && [[ $(echo -e "$VERSION_ID\n8.8"|sort -rV|head -n 1) == "8.8" ]]; then
+    yum update platform-python
```

Comment:
Should be `yum update -y platform-python`.

Reply:
Good catch. Thank you.

Reply:
Should add sudo before yum/dnf